bin/json_schemas/complete_config_schema.json

```diff
@@ -1454,11 +1430,25 @@
           "properties" : {
             "method" : {
               "type" : "string",
-              "not" : {"enum":["JUnitTestGrader", "EmmaInstrumentationGrader", "MultipleJUnitTestGrader",
-                       "EmmaCoverageReportGrader", "JaCoCoCoverageReportGrader", "DrMemoryGrader",
-                       "PacmanGrader", "searchToken", "intComparison", "diff", "fileExists",
-                       "warnIfNotEmpty", "warnIfEmpty", "errorIfNotEmpty", "errorIfEmpty", "ImageDiff",
-                       "custom_validator"]}
+              "not" : {
+                "enum": [
+                  "JUnitTestGrader",
+                  "MultipleJUnitTestGrader",
+                  "JaCoCoCoverageReportGrader",
+                  "DrMemoryGrader",
+                  "PacmanGrader",
+                  "searchToken",
+                  "intComparison",
+                  "diff",
+                  "fileExists",
+                  "warnIfNotEmpty",
+                  "warnIfEmpty",
+                  "errorIfNotEmpty",
+                  "errorIfEmpty",
+                  "ImageDiff",
+                  "custom_validator"
```

Comment:
Beyond the formatting change, I removed `EmmaInstrumentationGrader` and `EmmaCoverageReportGrader` from the list of allowed enums here.